src/libs/antares/study/parts/short-term-storage/series.cpp

```diff
@@ -221,11 +243,28 @@ static bool checkSize(const std::string& seriesFilename,
     return true;
 }
 
+static bool checkSize(const std::string& seriesFilename, const std::string& id, const TimeSeries& v)
+{
+    // for (auto col = 0; col < v.numberOfColumns(); ++col)
+    // {
+    if (v.timeSeries.height != HOURS_PER_YEAR)
+    {
+        logs.warning() << "Short-term storage " << id
+                       << " Invalid size for file: " << seriesFilename << ". Got "
+                       << v.timeSeries.height << " lines, expected " << HOURS_PER_YEAR;
+        return false;
+    }
+    // }
+
+    return true;
+}
+
 bool Series::validateSizes(const std::string& id, StudyVersion studyVersion) const
 {
     bool ret = checkSize("PMAX-injection.txt", id, maxInjectionModulation)
                && checkSize("PMAX-withdrawal.txt", id, maxWithdrawalModulation)
-               && checkSize("inflows.txt", id, inflows)
+               // TODO not needed
+               //    && checkSize("inflows.txt", id, inflows.series)
```

Comment:
We now expect `width >=1 && height == HOURS_PER_YEAR`

Reply:
the implementation of `bool TimeSeries::loadFromFile`  and `inline bool Matrix<T, ReadWriteT>::loadFromCSVFile` seems to force width = 1,  heigth = HOURS_PER_YEAR, but i can add the check